### PR TITLE
Accuracy checker support for bloomz-560m

### PR DESCRIPTION
**Pull request by sstrehlk**

### CODE REVIEW

File: tools/accuracy_checker/requirements-test.in [hunk 1]
```diff
@@ -1,6 +1,7 @@
 pytest>=5.0,<=7.0.1; python_version < '3.10'
 pytest==7.2.0; python_version >= '3.10'
 pytest-mock~=2.0
+scipy>=1.5.4
```
Comment (eaidova) [inline tools/accuracy_checker/requirements-test.in:4]: why??? it is requirements for accuracy checker unit tests, scipy do not used in tests

File: tools/accuracy_checker/openvino/tools/accuracy_checker/metrics/language_modeling.py [hunk 2]
```diff
@@ -15,6 +15,7 @@
 """
 
 import numpy as np
+from scipy.special import log_softmax
```
Comment (eaidova) [inline tools/accuracy_checker/openvino/tools/accuracy_checker/metrics/language_modeling.py:18]: please make it optional for users who do not want to use this metric, please see as other packages wrapper into UnsupportedPackage if not present as hard dependency

**Comment by eaidova** [comment]
@sstrehlk please updated readme files for introduced parameters and components

File: tools/accuracy_checker/requirements-test.in (hunk 1 above)
Comment (sstrehlk) [reply tools/accuracy_checker/requirements-test.in:4]: I added it to eliminate one of PR check fail

Comment (sstrehlk) [reply tools/accuracy_checker/requirements-test.in]: removed after making scipy optional.

File: tools/accuracy_checker/openvino/tools/accuracy_checker/metrics/language_modeling.py (hunk 2 above)
Comment (sstrehlk) [reply tools/accuracy_checker/openvino/tools/accuracy_checker/metrics/language_modeling.py]: done

**Review by eaidova — APPROVED** [review]

**Pull request merged**